src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -850,5 +852,154 @@ public void It_allows_TargetFrameworkVersion_to_be_capitalized()
                 .Should()
                 .Pass();
         }
+
+        [WindowsOnlyTheory]
+        [InlineData("true", "true")]
+        [InlineData("true", "false")]
+        [InlineData("false", "true")]
+        [InlineData("false", "false")]
+        public void It_places_package_pdb_and_xml_files_in_output_directory(string enableCopyDebugSymbolFilesFromPackages, string enableDocumentationFilesFromPackages)
+        {
+            var testProject = new TestProject()
+            {
+                Name = "DesktopUsingPackageWithPdbAndXml",
+                TargetFrameworks = ToolsetInfo.CurrentTargetFramework,
+                IsExe = true,
+            };
+
+            testProject.PackageReferences.Add(new TestPackageReference("Microsoft.Build", "17.3.1"));
+            
+            testProject.AdditionalProperties.Add("CopyDebugSymbolFilesFromPackages", enableCopyDebugSymbolFilesFromPackages);
+            testProject.AdditionalProperties.Add("CopyDocumentationFilesFromPackages", enableDocumentationFilesFromPackages);
+
+            TestAsset testAsset = _testAssetsManager.CreateTestProject(testProject, testProject.Name);
```

Comment:
You'll need to pass an identifier to make sure the different instances of the theory use different test paths:

```suggestion
            TestAsset testAsset = _testAssetsManager.CreateTestProject(testProject, testProject.Name, enableCopyDebugSymbolFilesFromPackages.ToString() + enableDocumentationFilesFromPackages.ToString());
```

Reply:
done in last commit